static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx

```diff
@@ -47,6 +49,8 @@ function renderValue(
 
 interface ProfileDetailsProps {
   profileGroup: ProfileGroup;
+  projectId: string;
+  transaction: Event | null;
```

Comment:
We should be more specific than just `Event` here

```suggestion
  transaction: EventTransaction | null;
```

Reply:
Had no idea we even had that tbh